akka-http-model/src/main/scala/akka/http/impl/util/JavaMapping.scala

```diff
@@ -66,7 +65,7 @@ private[http] trait JavaMapping[_J, _S] extends J2SMapping[_J] with S2JMapping[_
 
 /** INTERNAL API */
 @InternalApi
-private[http] object JavaMapping {
+private[http] trait JavaMappings {
```

Comment:
In an attempt to re-use the implicits defined here, I renamed it and converted to a trait, so the counter part in `akka-http-core` can extend it as well.